src/Components/MyListsHandler.cs

```diff
@@ -288,106 +315,192 @@ private void ResetSearchAndDisplayIngredients(TextBox searchBox, StackPanel ingr
+                Foreground = ingTextCol,
                 VerticalAlignment = VerticalAlignment.Center,
-                Margin = new Thickness(0, 0, 10, 0)
+                FontSize = ingTextFont,
+                FontWeight = FontWeights.Bold,
             };
-            DockPanel.SetDock(ingredientText, Dock.Left);
-            ingredientRow.Children.Add(ingredientText);
 
-            // Panel to hold delete and edit buttons together, aligned to the right
-            StackPanel buttonPanel = new StackPanel { Orientation = Orientation.Horizontal, HorizontalAlignment = HorizontalAlignment.Right };
+            // Ingredient type text 
+            TextBlock ingredientTypeText = new TextBlock
+            {
+                Text = $"{ingredient.GetIngType()}",
+                Foreground = ingTextCol,
+                VerticalAlignment = VerticalAlignment.Center,
+                FontSize = ingTextFont - 4,
+            };
 
-            // Create move button with text
-            Button moveButton = new Button
+            // Ingredient amount, unit text 
+            TextBlock ingredientAmountText = new TextBlock
             {
-                Content = "Move",
-                FontSize = 16,
-                Margin = new Thickness(5, 0, 0, 0),
-                Background = Brushes.White
+                Text = $"{ingredient.GetAmount()} {ingredient.GetUnit()}",
+                Foreground = ingTextCol,
+                VerticalAlignment = VerticalAlignment.Center,
+                FontSize = ingTextFont - 4,
             };
-            moveButton.Click += async (s, e) => await ShowMoveIngredientPopup(userList.GetListName(), ingredient);
 
-            // Add move button to the panel 
-            buttonPanel.Children.Add(moveButton);
+            DockPanel.SetDock(ingredientNameText, Dock.Top);
+            ingredientRow.Children.Add(ingredientNameText);
+            DockPanel.SetDock(ingredientTypeText, Dock.Top);
+            ingredientRow.Children.Add(ingredientTypeText);
+            DockPanel.SetDock(ingredientAmountText, Dock.Top);
+            ingredientRow.Children.Add(ingredientAmountText);
 
-            // Delete button (Trash icon)
+            // Panel to hold delete, edit, and move buttons
+            StackPanel buttonPanel = new StackPanel
+            {
+                Orientation = Orientation.Horizontal,
+                HorizontalAlignment = HorizontalAlignment.Right
+            };
+
+            // Move button
+            Button moveButton = new Button
+            {
+                Content = "\u21C4",
+                FontSize = buttonFont + 6,
+                FontWeight = FontWeights.Bold,
+                Margin = new Thickness(20, 10, 5, 5),
+                Background = Brushes.Transparent,
+                BorderBrush = Brushes.Transparent,
+                Foreground = buttonCol,
+                Cursor = Cursors.Hand,
+                ToolTip = "Move"
+            };
```

Comment:
Move button was cutoff in the actual screen.
![image](https://github.com/user-attachments/assets/d72b84a1-1f5e-418f-b3ba-ef6257e0921f)

Reply:
I think I may swap out the unicode with icons. The font and unicode representation looks diff on diff computers so this issue keeps popping up. Using an icon should get rid of this.